tests/Unit/Provider/TalkRatingProviderTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Test\Unit\Provider;
+
+use Mockery;
+use OpenCFP\Application;
+use OpenCFP\Domain\Services\Authentication;
+use OpenCFP\Domain\Services\TalkRating\OneToTenRating;
+use OpenCFP\Domain\Services\TalkRating\TalkRatingStrategy;
+use OpenCFP\Domain\Services\TalkRating\YesNoRating;
+use OpenCFP\Environment;
+use OpenCFP\Test\Helper\Faker\GeneratorTrait;
+
+/**
+ * @covers \OpenCFP\Provider\TalkRatingProvider
+ */
+class TalkRatingProviderTest extends \PHPUnit\Framework\TestCase
+{
+    use GeneratorTrait;
+
+    /**
+     * @dataProvider strategySettingsProvider
+     */
+    public function testItReturnsTheRightType($configInput, string $className)
+    {
+        //Set up our application object
+        $app                 = new Application(__DIR__ . '/../../..', Environment::testing());
```

Comment:
Giving that we are crossing multiple boundaries here, this looks more like an integration test to me.

Can we modify this test so we are just testing the `TalkRatingProvider` or convert it into an integration test?